runelite-client/src/main/java/net/runelite/client/plugins/mining/Rock.java

```diff
@@ -29,27 +29,8 @@
 import java.util.Map;
 import lombok.AccessLevel;
 import lombok.Getter;
-import static net.runelite.api.ObjectID.ROCKS_10943;
-import static net.runelite.api.ObjectID.ROCKS_11161;
-import static net.runelite.api.ObjectID.ROCKS_11360;
-import static net.runelite.api.ObjectID.ROCKS_11361;
-import static net.runelite.api.ObjectID.ROCKS_11364;
-import static net.runelite.api.ObjectID.ROCKS_11365;
-import static net.runelite.api.ObjectID.ROCKS_11366;
-import static net.runelite.api.ObjectID.ROCKS_11367;
-import static net.runelite.api.ObjectID.ROCKS_11368;
-import static net.runelite.api.ObjectID.ROCKS_11369;
-import static net.runelite.api.ObjectID.ROCKS_11370;
-import static net.runelite.api.ObjectID.ROCKS_11371;
-import static net.runelite.api.ObjectID.ROCKS_11372;
-import static net.runelite.api.ObjectID.ROCKS_11373;
-import static net.runelite.api.ObjectID.ROCKS_11374;
-import static net.runelite.api.ObjectID.ROCKS_11375;
-import static net.runelite.api.ObjectID.ROCKS_11376;
-import static net.runelite.api.ObjectID.ROCKS_11377;
-import static net.runelite.api.ObjectID.ROCKS_11386;
-import static net.runelite.api.ObjectID.ROCKS_11387;
-import static net.runelite.api.ObjectID.ASH_PILE;
+
```

Comment:
remove this empty line. Also, import RL codestyle to your IDE and reorganize imports: https://github.com/runelite/runelite/wiki/Code-Conventions